project/MicroServiceBuild.scala

```diff
@@ -10,21 +10,19 @@ object MicroServiceBuild extends Build with MicroService {
 
   val compile = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-play-25" % "4.0.0",
-    "uk.gov.hmrc" %% "play-ui" % "7.26.0-play-25",
-    "uk.gov.hmrc" %% "domain" % "5.2.0",
-    "uk.gov.hmrc" %% "agent-mtd-identifiers" % "0.12.0",
+    "uk.gov.hmrc" %% "bootstrap-play-25" % "4.8.0",
+    "uk.gov.hmrc" %% "domain" % "5.3.0",
+    "uk.gov.hmrc" %% "agent-mtd-identifiers" % "0.13.0",
     "uk.gov.hmrc" %% "tax-year" % "0.4.0",
-    "uk.gov.hmrc" %% "auth-client" % "2.17.0-play-25",
+    "uk.gov.hmrc" %% "auth-client" % "2.19.0-play-25",
     "uk.gov.hmrc" %% "mongo-caching" % "5.6.0"
   )
 
   def test(scope: String = "test,it") = Seq(
-    "uk.gov.hmrc" %% "hmrctest" % "3.3.0" % scope,
+    "uk.gov.hmrc" %% "hmrctest" % "3.4.0-play-25" % scope,
     "org.pegdown" % "pegdown" % "1.6.0" % scope,
-    "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
     "uk.gov.hmrc" %% "reactivemongo-test" % "3.1.0" % scope,
```

Comment:
Latest reactivemongo-test is https://github.com/hmrc/reactivemongo-test/releases/tag/v4.7.0